genestack_client/genestack_shell.py

```diff
@@ -298,14 +300,16 @@ def do_EOF(self, line):
 
     def process_command(self, command, argument_list, shell=False):
         """
-        Run a command with arguments.
+        Run a command with arguments, returns exit code.
```

Comment:
This description is still useless: what exit codes can it return? who returns this exit code, the command or the method? do these exit codes have any meanings?